app/jobs/reports/irs_weekly_summary_report.rb

```diff
@@ -0,0 +1,53 @@
+require 'csv'
+
+module Reports
+    class IrsWeeklySummaryReport < BaseReport
```

Comment:
this is indented by 4, but usually we indent by 2 (same for the whole file)
```suggestion
  class IrsWeeklySummaryReport < BaseReport
```